doc/lua_api.txt

```diff
@@ -5343,7 +5343,7 @@ Defaults for the `on_punch` and `on_dig` node definition callbacks
 
 * `minetest.node_punch(pos, node, puncher, pointed_thing)`
     * Calls functions registered by `minetest.register_on_punchnode()`
-* `minetest.node_dig(pos, node, digger)`
+* `minetest.node_dig(pos, node, digger, pointed_thing)`
```

Comment:
You have to make this new parameter optional to not break api (= accept `nil` for `pointed_thing`).
```suggestion
* `minetest.node_dig(pos, node, digger[, pointed_thing])`
```
Edit: Seems like digger can also be `nil`, which is where you can see that minetest's documentation is not existent at some places.

Reply:
Yes, digger can be nil. I'll write `minetest.node_dig(pos, node[, digger[, pointed_thing]])`.